study-builder/studies/testboston/followup-survey.conf

```diff
@@ -0,0 +1,163 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": ${id.act.followup},
+  "versionTag": "v1",
+  "displayOrder": 4,
+  "writeOnce": false,
+  "editTimeoutSec": 864000,   # 10 days
```

Comment:
This is the longitudinal activity that expires in 10 days. A new instance of this is created every time we do a new kit.